Operational-Procedures/Tutorials/testing-a-csc.rst

```diff
@@ -0,0 +1,156 @@
+###########################
+Testing A CSC At The Summit
+###########################
+
+Overview
+^^^^^^^^
+
+This document decribes how to test a CSC, that is still under development, at the summit.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+- Announcements are to be made in the summit-testing-annoucements Slack channel
```

Comment:
I hate to tell you but they just renamed the channel to `summit-announce`

Reply:
Yeah I noticed that. I changed it, also in the `announce-component-usage.rst` file.